src/Symfony/Contracts/Translation/InvalidArgumentExceptionInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Contracts\Translation;
+
+/**
+ * Exception interface for all exceptions thrown by the component.
```

Comment:
Looks wrong to me

Reply:
But anyway, why do we need this interface in the first place?

Reply:
I removed `InvalidArgumentExceptionInterface` and updated `TranslationInterface` with `@throws \InvalidArgumentException` instead.